test/test_case_5_verify_that_a_users_status_can_be_enabled_or_disabled.py

```diff
@@ -18,5 +18,3 @@ def test_case_5_verify_that_user_status_can_be_enabled_or_disabled(app):
     app.orangeHrm.popUp.click_on_username_field()
     app.orangeHrm.popUp.add_filter_username('NotAnAdmin')
     app.orangeHrm.popUp.click_on_search()
-
-# I have no idea how to confirm empty results page and the 'No Records Found' message :(
```

Comment:
filter_no_records_message = '//div[text()="No Records Found"]'
filtered_usernames = 'tbody td:nth-child(2) span'


def make_sure_that_user_not_found(self):
    self.step.specified_element_is_not_present(self.filter_no_records_message, 5)
    return self.step.specified_element_is_present(self.filtered_usernames)

Locate this to the right place in popUp component